CMakePresets.json

```diff
@@ -0,0 +1,108 @@
+{
+  "version": 3,
+  "configurePresets": [
+    {
+      "name": "windows-base",
+      "description": "Target Windows with the Visual Studio development environment.",
+      "hidden": true,
+      "generator": "Visual Studio 17 2022",
+      //"generator": "Ninja",
+      "binaryDir": "${sourceDir}/msvc.build/${presetName}",
+      "installDir": "${sourceDir}/msvc.install/${presetName}",
+      "cacheVariables": {
+        "CMAKE_C_COMPILER": "cl.exe",
+        "CMAKE_CXX_COMPILER": "cl.exe",
+        "CMAKE_WARN_DEPRECATED": "FALSE",
+        "CMAKE_PREFIX_PATH": "$env{PATH};${sourceDir}\\dependencies"
+      },
+      "condition": {
+        "type": "equals",
+        "lhs": "${hostSystemName}",
+        "rhs": "Windows"
+      }
+    },
+    {
+      "name": "x64-debug",
+      "displayName": "x64 Debug",
+      "description": "Target Windows (64-bit) with the Visual Studio development environment. (Debug)",
+      "inherits": "windows-base",
+      "architecture": {
+        "value": "x64",
+        "strategy": "external"
+      },
+      "cacheVariables": {
+        "CMAKE_LIBRARY_PATH": "${sourceDir}\\dependencies\\libx64",
+        "CMAKE_BUILD_TYPE": "Debug",
+        "MUSE_COMPILE_BUILD_64": "ON"
```

Comment:
Why are `CMAKE_LIBRARY_PATH` and `MUSE_COMPILE_BUILD_64` only here for "Debug"? I think `MUSE_COMPILE_BUILD_64` is unnecessary anyway, not sure about the other one.